src/sql/workbench/contrib/resourceViewer/browser/resourceViewerTable.ts

```diff
@@ -93,4 +102,19 @@ export class ResourceViewerTable extends Disposable {
 		}
 		return value;
 	}
+
+	private async onTableClick(event: ITableMouseEvent): Promise<void> {
+		const column = this._resourceViewerTable.columns[event.cell.cell];
+		if (column) {
+			const row = this._dataView.getItem(event.cell.row);
+			const value = row[column.field];
+			if (isHyperlinkCellValue(value)) {
+				if (isString(value.linkOrCommand)) {
+					await this._openerService.open(value.linkOrCommand);
+				} else {
```

Comment:
if the value is undefined, it will also fall into the else block here, right?

Reply:
If value is undefined it won't even go into the isHyperlinkCellValue block so will just do nothing

Reply:
yeah, that should be fine with the way you implemented isHyperlinkCellValue